.ci/bump-opbeans-node.groovy

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+opbeansPipeline()
```

Comment:
@v1v Probably don't want this line, right?

Reply:
Good catch ! :)